WorkflowTesting/Tests/WorkflowRenderTesterFailureTests.swift

```diff
@@ -340,6 +340,18 @@ final class WorkflowRenderTesterFailureTests: XCTestCase {
             }
         }
     }
+
+    func test_assertState() {
+        let result = TestWorkflow()
+            .renderTester(initialState: .idle)
+            .render { _ in }
+
```

Comment:
Shall we test that `assertState` can succeed as well?

I'm not sure why we haven't done so for the other, existing assert/verify methods 🤔

Reply:
We are already doing that in `WorkflowRenderTesterTests/test_childWorkflowOutput` for both this new method and `verifyState`